src/main/java/uk/org/tombolo/execution/DataExportEngine.java

```diff
@@ -52,7 +52,9 @@ public void execute(DataExportRecipe dataExportSpec, Writer writer, ImporterMatc
 
 		// Import datasources that are in the global dataset specification
 		for (DatasourceRecipe datasourceSpec : dataExportSpec.getDataset().getDatasources()) {
-			importDatasource(forceImports, datasourceSpec, dataExportSpec.getDataset().getSubjects());
+			if (!datasourceSpec.getImporterClass().isEmpty()) {
```

Comment:
When is this empty?

Reply:
in just one situation, if importer is built in python, and user has only one datasource, then while building recipe user would pass importer class and datasource id as blank. As importer has already been executed from python side